.travis.yml

```diff
@@ -0,0 +1,45 @@
+dist: trusty
+
+cache:
+  apt: true
+  directories:
+    # Cache directory for older Composer versions.
+    - $HOME/.composer/cache/files
+    # Cache directory for more recent Composer versions.
+    - $HOME/.cache/composer/files
+
+language:
+  - php
+
+php:
```

Comment:
Don't need this section, as the PHP versions are in the `matrix`, instead. It's just causing each PHP version to be run twice on Travis.